protocol/driver/driver.go

```diff
@@ -23,8 +23,8 @@ type Driver struct {
 	Version string
 	// Build identifier.
 	Build string
-	// UASTParserBuilder creates a ASTParser.
-	UASTParserBuilder UASTParserBuilder
+	// ParserBuilder creates a ASTParser.
```

Comment:
Update the comment too (which was wrong anyway since it was an UASTParser, now a Parser).